Sandbox/SandboxUITests/SignupLoginSuccess.swift

```diff
@@ -0,0 +1,112 @@
+import XCTest
+
+extension ClosedRange where Element: Hashable {
+    func random(without excluded:[Element]) -> Element {
+        let valid = Set(self).subtracting(Set(excluded))
+        let random = Int(arc4random_uniform(UInt32(valid.count)))
+        return Array(valid)[random]
+    }
+}
+
+class SignupLoginSuccess: XCTestCase {
+    
+    var uuid = UUID().uuidString
+    let rndPswd = "P@ssw0rd"
+    var newPhoneNumber = String()
+    
+    override func setUpWithError() throws {
+        
+        uuid += "@testaccount.io"
+        newPhoneNumber = randomPhoneNumber()
+        continueAfterFailure = false
+        
+    }
+    
+    override func tearDownWithError() throws {
+    }
+    
+    func testSignupLogin() throws {
+        
+        let app = XCUIApplication()
+        app.launch()
+        XCUIApplication().tables.staticTexts["Signup with password"].tap()
+        
+        // Test Signup with email
+        let emailTextField = app.textFields["email"]
+        let passwordTextField = app.textFields["password"]
+        
+        emailTextField.tap()
+        emailTextField.typeText(uuid)
+        sleep(1)
```

Comment:
Potentially stupid question as I'm not familiar at all with the whole testing story on IOS: is it possible to use async expectations to avoid all those calls to `sleep` ? I'm talking about [this](https://developer.apple.com/documentation/xctest/xctestcase) (section _Creating Asynchronous Test Expectations_.)